staging/src/k8s.io/kubectl/pkg/describe/describe_test.go

```diff
@@ -2844,19 +2847,21 @@ func TestDescribeDeployment(t *testing.T) {
 func TestDescribeJob(t *testing.T) {
 	indexedCompletion := batchv1.IndexedCompletion
 	cases := map[string]struct {
-		job                  *batchv1.Job
-		wantCompletedIndexes string
+		job                *batchv1.Job
+		expectedElements   []string
+		unexpectedElements []string
```

Comment:
To be more consistent with the other tests, better rename `expectedElements` to `wantElements` and `unexpectedElements` to `dontWantElements` since they all use these prefixes 👍🏻

Reply:
Ivan and I discussed about this and we agreed to leave it as it is since in this test file the `(un)expected` prefix is the most used 👍🏻

Reply:
Done.